provider/aws/lambda/autoscale/handler.go

```diff
@@ -314,6 +314,13 @@ func ci(ii ...*int64) int64 {
 	return 0
 }
 
+func log(format string, a ...interface{}) {
```

Comment:
Make two functions, `log` and `debug`.

`debug` should only write out if `DEBUG=true`

Change all other Printfs to use log which should always print